tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java

```diff
@@ -396,14 +404,12 @@ public void parentDirectoryPermissionForUfsTest() throws IOException {
   }
 
   /**
-   * Tests for the permission of object storage UFS files.
+   * Tests for the permission of S3 UFS files.
    */
   @Test
-  public void objectStoragePermissionTest() throws Exception {
-    if (!(sUfs instanceof S3UnderFileSystem) && !(sUfs instanceof S3AUnderFileSystem)
-        && !(sUfs instanceof SwiftUnderFileSystem) && !(sUfs instanceof GCSUnderFileSystem)
-        && !(sUfs instanceof OSSUnderFileSystem)) {
-      // Only test object storage UFSs.
+  public void s3PermissionTest() throws Exception {
+    if (!(sUfs instanceof S3UnderFileSystem) && !(sUfs instanceof S3AUnderFileSystem)) {
```

Comment:
Could we use [Assume](http://junit.sourceforge.net/javadoc/org/junit/Assume.html) for this?